app/publish/publish.go

```diff
@@ -127,7 +150,18 @@ func (h UploadHandler) CreateFile(accountID string, origFilename string) (*os.Fi
 }
 
 func (h UploadHandler) preparePath(accountID string) (string, error) {
-	path := path.Join(h.uploadPath, accountID)
+	path := path.Join(h.UploadPath, accountID)
 	err := os.MkdirAll(path, os.ModePerm)
 	return path, err
 }
+
+// CanHandle checks if http.Request contains POSTed data in an accepted format.
+// Supposed to be used in gorilla mux router MatcherFunc.
+func (h UploadHandler) CanHandle(r *http.Request, _ *mux.RouteMatch) bool {
```

Comment:
A better name would be `HasAcceptableFormat`

Reply:
There is a point but I think I'd rather keep the name describing _the what_ (we're checking if we can handle the request) instead of disclosing _the how_ (we're checking the format of the supplied request because..?). Hope it makes sense :)